internal/postprocessor/main.go

```diff
@@ -316,9 +316,9 @@ func (p *postProcessor) UpdateSnippetsMetadata() error {
 		// first path segment is the second element.
 		moduleName := strings.Split(clientRelPath, "/")[1]
 		if moduleName == "" {
-			return errors.New("UpdateSnippetsMetadata: moduleName is empty")
+			return fmt.Errorf("unable to parse module name for %v", clientRelPath)
 		}
-		log.Println("UpdateSnippetsMetadata: ", clientRelPath)
+		log.Println(clientRelPath)
```

Comment:
nit: maybe a little more context here, like add the word processing or something?

Reply:
I moved the statement after the filtering for `debugger` etc, and changed the module name to the metadata file glob path. Sample output when run locally in `internal/postprocessor`:

```sh
2023/05/01 12:08:26 updating snippets metadata
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/accessapproval/apiv1/snippet_metadata.*.json
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/accesscontextmanager/apiv1/snippet_metadata.*.json
...
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/compute/apiv1/snippet_metadata.*.json
2023/05/01 12:08:26 skipping, file not found with glob:  ../../internal/generated/snippets/confidentialcomputing/apiv1/snippet_metadata.*.json
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/confidentialcomputing/apiv1alpha1/snippet_metadata.*.json
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/contactcenterinsights/apiv1/snippet_metadata.*.json
2023/05/01 12:08:26 updating  ../../internal/generated/snippets/container/apiv1/snippet_metadata.*.json
```